libs/alsa-lib/Makefile

```diff
@@ -13,14 +13,15 @@ PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=ftp://ftp.alsa-project.org/pub/lib/ \
-		http://distfiles.gentoo.org/distfiles/
+		https://distfiles.gentoo.org/distfiles/
 
 PKG_HASH:=5f2cd274b272cae0d0d111e8a9e363f08783329157e8dd68b3de0c096de6d724
 PKG_MAINTAINER:=Ted Hess <thess@kitschensync.net>, \
 		Peter Wagner <tripolar@gmx.at>
 
 PKG_LICENSE:=LGPLv2.1 GPLv2
 PKG_LICENSE_FILES:=COPYING aserver/COPYING
+PKG_CPE_ID:=cpe:/o:alsa:alsa-lib
```

Comment:
why do you use cpe:/o: and not cpe:/a: ? to be honest i didn't read about it but a grep gave me

> admin/htop/Makefile:PKG_CPE_ID:=cpe:/a:htop:htop
> admin/monit/Makefile:PKG_CPE_ID:=cpe:/a:tildeslash:monit
> libs/libxml2/Makefile:PKG_CPE_ID:=cpe:/a:xmlsoft:libxml2
> libs/tiff/Makefile:PKG_CPE_ID:=cpe:/a:libtiff:libtiff
> utils/unrar/Makefile:PKG_CPE_ID:=cpe:/a:rarlab:unrar

Reply:
I don't think this is right either. ```cpe:/o:``` is for operating system (like linux kernel). Do you have a definitive reference URL I can check.

Reply:
I used this page as a reference: https://nvd.nist.gov/vuln/detail/CVE-2005-0087

The o might stand for Oval based on that page. Not sure.

Reply:
There is no "alsa" anything in the CPE dictionary. I think you found a typo or error. Try to query the database here: https://nvd.nist.gov/products/cpe/search